src/DataBox/DataBox/Common/Cmdlets/StopDataBoxJobCmdlet.cs

```diff
@@ -48,6 +54,9 @@ public class StopDataBoxJob : AzureDataBoxCmdletBase
         [Parameter(Mandatory = false)]
         public SwitchParameter PassThru;
 
+        [Parameter(Mandatory = false)]
+        public SwitchParameter Force;
```

Comment:
This should not have a Force parameter

Reply:
Other Stop cmdlets from other modules also have a Force option. Ex: Stop-AzureVM (https://docs.microsoft.com/en-us/powershell/module/servicemanagement/azure/stop-azurevm?view=azuresmps-4.0.0) . Stop-AzDataBoxJob is used to cancel a job so it needs a optional Force option.